crates/cairo-profiler/src/trace_reader/function_trace_builder.rs

```diff
@@ -15,12 +16,12 @@ use trace_data::TraceEntry;
 mod function_stack_trace;
 
 pub struct FunctionLevelProfilingInfo {
-    pub functions_stack_traces: Vec<FunctionStackTrace>,
+    pub functions_traces: Vec<FunctionCallTrace>,
     pub header_steps: Steps,
 }
 
-pub struct FunctionStackTrace {
-    pub stack_trace: Vec<FunctionName>,
+pub struct FunctionCallTrace {
+    pub call_trace: Vec<Function>,
     pub steps: Steps,
 }
```

Comment:
Isn't this a sample?

Reply:
True, `FunctionSample` is better

Reply:
why `Function` prefix? `Function` is only a name here the rest is a sample.

Reply:
Because `Sample` is another, more general struct

Reply:
Will change it